generated/nidigital/nidigital/unit_tests/test_nidigital.py

```diff
@@ -29,6 +29,12 @@ def setup_method(self, method):
         self.patched_library.niDigital_UnlockSession.side_effect = self.side_effects_helper.niDigital_UnlockSession
         self.side_effects_helper['UnlockSession']['callerHasLock'] = True
 
+        # For trying to set `_all_channels_in_session` in the Session constructor
+        self.patched_library.niDigital_GetAttributeViInt32.side_effect = self.side_effects_helper.niDigital_GetAttributeViInt32
+        self.side_effects_helper['GetAttributeViInt32']['value'] = 1  # channel_count
+        self.patched_library.niDigital_GetChannelNameFromString.side_effect = self.side_effects_helper.niDigital_GetChannelNameFromString
+        self.side_effects_helper['GetChannelNameFromString']['names'] = '0'  # get_channel_names()
+
```

Comment:
Is there anywhere in our testing where we confirm that `_all_channels_in_session` is set correctly?
Naturally it would belong in `nifake` but I understand that would require metadata changes.

Reply:
Thinking about this a bit more, I think it's fine. We have tests in place for the public facing behavior in `nidcpower` and this is what matters.